src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -26,7 +32,7 @@ object PlayBuildBase extends AutoPlugin {
     val playBuildPromoteBintray = settingKey[Boolean]("Whether a Bintray promotion should be done on release")
     val playBuildPromoteSonatype = settingKey[Boolean]("Whether a Sonatype promotion should be done on release")
     val playCrossBuildRootProject = settingKey[Boolean]("Whether the root project should be cross built or not")
-    val playBuildRepoName = settingKey[String]("The name of the playframework GitHub repository")
+    val playBuildRepoName = settingKey[String]("The name of the PlayFramework GitHub repository")
```

Comment:
If you wanted to make this more clear, you could probably say "The name of the repository in the playframework GitHub organization."